package.json

```diff
@@ -26,7 +26,7 @@
     "@angular/platform-browser": "~10.0.11",
     "@angular/platform-browser-dynamic": "~10.0.11",
     "@angular/router": "~10.0.11",
-    "@stoplight/elements-web-components": "6.0.0-beta.163",
+    "@stoplight/elements-web-components": "^6.0.0-beta.166",
```

Comment:
Let's bring it back to `beta`

Reply:
Unless there's a reason for a specific version of course

Reply:
I'd rather keep it like this, and will set up dependabot to auto-trigger a PR that bumps elements whenever we release a new version. That way we keep the examples up to date immediately and it also runs a sanity check (build) to make sure we didn't break anything.